README.md

```diff
@@ -45,30 +44,25 @@ Graphics that will help you and your team better understand the concepts and how
 Try the interactive infographics on the [website](http://habitat.sh#reference-diagram)!
 
 ### How Habitat Works
-* [Architecture Overview](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-architecture-overview.png)
-* [Initial Package Build Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-initial-package-build-flow.png)
-* [Application Rebuild Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-application-rebuild-flow.png)
-* [Dependency Update Flow](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-dependency-update-flow.png)
-* [Promote Packages Through Channels](https://github.com/habitat-sh/habitat/raw/master/www/source/images/infographics/habitat-promote-packages-through-channels.png)
+* [Architecture Overview](images/habitat-architecture-overview.png)
+* [Initial Package Build Flow](images/habitat-initial-package-build-flow.png)
+* [Application Rebuild Flow](images/habitat-application-rebuild-flow.png)
+* [Dependency Update Flow](images/habitat-dependency-update-flow.png)
+* [Promote Packages Through Channels](images/habitat-promote-packages-through-channels.png)
```

Comment:
All of the above links should be "raw" links otherwise the image will appear in the github view and be cut off. This probably means you need to commit the images before this PR is merged so you have access to those raw links. Go ahead and create a separate pr just for these images and we can get that merged asap.

Reply:
Will do.